src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1542,3 +1542,167 @@
+    examples:
+        - name: remove the hybrid connection from the functionapp
+          text: >
+            az functionapp hybrid-connection remove -g [myResourceGroup] -n [myAppName] -s [slotName]
+            --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+"""
+
+helps['webapp hybrid-connection'] = """
+    type: group
+    short-summary: commands that list, add, or remove hybrid connections from a sepcific app
+"""
+
+helps['webapp hybrid-connection list'] = """
+    type: command
+    short-summary: lists all the hybrid connections set on the specified app
+    examples:
+        - name: list the hybrid connections
+          text: >
+              az webapp hybrid-connection list -g [myResourceGroup} -n {myAppName] -s [slotName]
+"""
+
+helps['webapp hybrid-connection add'] = """
+    type: command
+    short-summary: adds an existing hybrid connection to the specified app
+    examples:
+        - name: add a hybrid connection to the app
+          text: >
+            az webapp hybrid-connection add -g [myResourceGroup] -n [myAppName] -s [slotName]
+            --namespace [hybridConnectionNamespaceName] --hybrid-connection [hybridConnectionName]
+"""
+
+helps['webapp hybrid-connection remove'] = """
+    type: command
+    short-summary: removes an existing hybrid connection from the specified app
+    examples:
+        - name: remove the hybrid connection from the app
+          text: >
+            az webapp hybrid-connection remove -g [myResourceGroup] -n [myAppName] -s [slotName]
+            --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+"""
+
+helps['appservice hybrid-connection'] = """
+    type: group
+    short-summary: commands that interact with hybrid connections on an app service level - existing command sets which key each app uses to connect to the hybric connection
+"""
+
+helps['appservice hybrid-connection set-key'] = """
+    type: command
+    short-summary: specifies which key all the apps using a hybrid connection should use when connecting to it
+    examples:
+        - name: set the key the apps should use
+          text: >
+            az appservice hybrid-connection set-key --plan [hybridConnectionAppServicePlan] -g [appServicePlanResourceGroup]
+            --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+            --key-type ["primary" | "secondary"]
+"""
+
+helps['appservice vnet-integration'] = """
+    type: group
+    short-summary: commands that operate on virtual network integrations in a specified app service plan
+"""
+
+helps['appservice vnet-integration list'] = """
+    type: command
+    short-summary: lists all the virtual networks in a specified app service plan
+    examples:
+        - name: list the virtual network integrations
+          text: >
+            az appservice vnet-integration list -g [myResourceGroup] --plan [MyAspName]
+"""
```

Comment:
will we have `remove`?

Reply:
Also will we have a `show` command, or `vnet-integration` is not any kind of resource?